### ISSUE TITLE
LICENSE being installed to the wrong path

### ISSUE THREAD

**Issue by algorythmic** [c0fec0de/anytree#139]
As specified currently in setup.py with

```python
config['data_files'] = [('', ['LICENSE'])]
```

this gets installed right into `sys.prefix` (or causes an error with homebrew python):

```
$ pip install anytree
Collecting anytree
  Using cached anytree-2.8.0-py2.py3-none-any.whl (41 kB)
Requirement already satisfied: six>=1.9.0 in /usr/local/lib/python3.8/site-packages (from anytree) (1.15.0)
Installing collected packages: anytree
ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/local/LICENSE'
Consider using the `--user` option or check the permissions.
```

Per https://github.com/pypa/setuptools/pull/1536 this should not be necessary, rather `license_file` can be specified instead.

**Comment by c0fec0de** [comment]
will fix it

**Comment by iqbal-lab** [comment]
Would be v grateful for this fix!

**Comment by teauxfu** [comment]
related #150 to be resolved by #172

**Comment by paper42** [comment]
@c0fec0de this issue can be closed, because #165 was merged
